canvas_course_admin_tools/settings/local.py

```diff
@@ -29,6 +29,12 @@
 dictConfig(LOGGING)
 
 SELENIUM_CONFIG = {
+    'class_roster': {
+        'course_link': 'https://canvas.dev.tlt.harvard.edu/courses/3787'
```

Comment:
Good point. I went this way because I thought it would be easier than do URL join in the code, but you're right - it makes for cleaner and more consistent code.  Fixing this -- thanks!